esp/src/eclwatch/PackageMapPartsWidget.js

```diff
@@ -93,9 +93,8 @@ define([
                     var addPartInput = context.getFilter();
                     WsPackageMaps.AddPartToPackageMap({
                         request: {
-                            Target: addPartInput.AddPartsTarget,
-                            Process: addPartInput.Process,
-                            PackageMap: addPartInput.PackageMap,
+                            Target: context.params.target,
+                            PackageMap: context.params.packageMap.split('::')[1],
```

Comment:
@miguelvazq to clarify the id will usually have the target prepended... but there is a way to create one from command line that doesn't have that... so you should probably trim it off if its there, but use the whole string if it's not there.